packages/loopring_v3/contracts/impl/libexchange/ExchangeGenesis.sol

```diff
@@ -52,7 +58,7 @@ library ExchangeGenesis
         S.blocks.push(ExchangeData.BlockInfo(bytes32(0)));
 
         // Get the protocol fees for this exchange
-        S.protocolFeeData.syncedAt = uint32(0);
+        S.protocolFeeData.syncedAt = uint32(now);
```

Comment:
`0` is probably better because we only set the protocol fees to the max value here, so it would take some time before the actual values are used when blocks are being submitted immediately after the exchange is created. Probably very unlikely because it will take some time before blocks are submitted after the exchange is created, but I think it makes sense to check the actual values the first time a block is submitted to get the recent values.